.github/workflows/test-deployer.yaml

```
@@ -0,0 +1,30 @@
+name: Run the deployer's tests
+
+on:
+  push:
+    paths:
+      - "**/deployer/**"
+      - "tests/**"
+  pull_request:
+    paths:
+      - "**/deployer/**"
+      - "tests/**"
+
+jobs:
+  test-deployer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions:setup-python@v3
+        with:
+          python-version: 3.9
```

```suggestion
          python-version: "3.9"
```

Without quotes for 3.10, it will become `3.1` and things fail. Adding quotes directly to avoid a future issue.